src/Learning/MODL/MODL.cpp

```diff
@@ -4,6 +4,27 @@
 
 #include "MODL.h"
 
+// Debogage sous Windows Visual C++ 2022 (bug https://github.com/microsoft/vscode-cpptools/issues/8084)
+// Choix en dur du repertoire de lancement
+void SetWindowsDebugDir(const ALString& sDatasetFamily, const ALString& sDataset)
+{
+#ifdef _WIN32
+	ALString sUserRootPath;
+	int nRet;
+
+	// A parametrer pour chaque utilisateur
+	sUserRootPath = "D:/Users/miib6422/Documents/boullema/LearningTest/TestKhiops/";
```

Comment:
Why not an environment variable ? This would avoid a local path in the code.

Reply:
C'est évidemment temporaire, et tellement simple à changer a la demande. 
Plutôt que de mettre en place une variable d'environnement, à documenter, configurer, modifier si nécessaire...
Et on peut espérer que ce bug sera un jour résolu